src/isomorphic/modern/class/ReactBaseClasses.js

```diff
@@ -21,7 +21,28 @@ var warning = require('fbjs/lib/warning');
 /**
  * Base class helpers for the updating state of a component.
  */
+
+const addUpdaterMutationWarn = context => {
```

Comment:
Right 👍 ! I was using `const` instead of `var` which doesn't get hoisted to the top